tests/test_cli_delete.py

```diff
@@ -0,0 +1,66 @@
+import json
+import pytest
+
+from click.testing import CliRunner
+from unittest import mock
+
+from mapbox_tilesets.scripts.cli import delete
+from mapbox_tilesets.errors import TilesetsError
+
+
+class MockResponse:
+    def __init__(self, mock_json, status_code):
+        self.text = json.dumps(mock_json)
+        print(self.text)
+        self._json = mock_json
+        self.status_code = status_code
+
+    def MockResponse(self):
+        return self
+
+
+@pytest.mark.usefixtures("token_environ")
+@mock.patch("requests.delete")
+def test_cli_delete(mock_request_delete):
+    runner = CliRunner()
+
+    # sends expected request
+    mock_request_delete.return_value = MockResponse("", status_code=200)
+    result = runner.invoke(delete, ["test.id"], input="y")
```

Comment:
Could you add a test where the input is `n` to the confirmation prompt?

Reply:
Yep! Added in https://github.com/mapbox/tilesets-cli/pull/61/commits/4af26eb544a51a7f5b7df78e07a07e7e35dd7c68